src/payment/payment.module.ts

```diff
@@ -42,6 +42,11 @@ export type PaymentModuleConfig = {
   logger?: Logger;
 };
 
+export interface PayerDetails {
+  address: string;
+  platform: string;
```

Comment:
It looks like we could model `address`, `platform`, `driver` under the same hood? Just a moment ago we had this class - `PaymentPlatform`. I can see it's in the codebase but it's not used. Perhaps we can bundle all of the "payer/payee" information under a single type.

Reply:
Good idea, I changed the name to `PayerDetails` so it doesnt clash with what payment platform actually is